default.nix

```diff
@@ -69,12 +69,17 @@ let
     sources = localLib.sources;
     bridgeTable = {
       jormungandr = self.callPackage ./nix/jormungandr-bridge.nix {};
-      cardano = self.callPackage ./nix/cardano-bridge.nix {};
+      cardano = self.callPackage ./nix/cardano-bridge.nix {
+        cardano-wallet = if self.launcherConfigs.launcherConfig.nodeConfig.kind == "byron"
+                         then self.cardano-wallet.cardano-wallet-byron
+                         else self.cardano-wallet.cardano-wallet-shelley;
+        cardanoWalletPkgs = self.cardano-wallet.pkgs;
+      };
```

Comment:
you could also potentially do `cardano-byron` and `cardano-shelley` in this table